ci-operator/step-registry/openshift-pipelines-test/load-test/openshift-pipelines-test-load-test-commands.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+TEST=$(cat /usr/local/ci-secrets/redhat-appstudio-load-test/quay-oauth-user)
+OPENSHIFT_API="$(yq e '.clusters[0].cluster.server' "$KUBECONFIG")"
+OPENSHIFT_USERNAME="kubeadmin"
+
+export OPENSHIFT_PASSWORD
+export BYOC_KUBECONFIG
+
+export TEST
+export OPENSHIFT_API
+export OPENSHIFT_USERNAME
+
+echo -e "[INFO] Start tests"
+
+yq -i 'del(.clusters[].cluster.certificate-authority-data) | .clusters[].cluster.insecure-skip-tls-verify=true' "$KUBECONFIG"
+if [[ -s "$KUBEADMIN_PASSWORD_FILE" ]]; then
+    OPENSHIFT_PASSWORD="$(cat "$KUBEADMIN_PASSWORD_FILE")"
+elif [[ -s "${SHARED_DIR}/kubeadmin-password" ]]; then
+    # Recommendation from hypershift qe team in slack channel..
+    OPENSHIFT_PASSWORD="$(cat "${SHARED_DIR}/kubeadmin-password")"
+else
+    echo "Kubeadmin password file is empty... Aborting job"
+    exit 1
+fi
+
+timeout --foreground 5m bash <<-"EOF"
+    while ! oc login "$OPENSHIFT_API" -u "$OPENSHIFT_USERNAME" -p "$OPENSHIFT_PASSWORD" --insecure-skip-tls-verify=true; do
+            sleep 20
+    done
+EOF
+if [ $? -ne 0 ]; then
+    echo "Timed out waiting for login"
+    exit 1
+fi
+
+# Define a new environment for BYOC pointing to a kubeconfig with token. RHTAP environments only supports kubeconfig with token:
+# See: https://issues.redhat.com/browse/GITOPSRVCE-554
+BYOC_KUBECONFIG="/tmp/token-kubeconfig"
+cp "$KUBECONFIG" "$BYOC_KUBECONFIG"
+if [[ -s "$BYOC_KUBECONFIG" ]]; then
+    echo -e "byoc kubeconfig exists!"
+else
+    echo "Kubeconfig not exists in $BYOC_KUBECONFIG... Aborting job"
+    exit 1
+fi
+
+cd "$(mktemp -d)"
+
+git clone --branch main "https://github.com/openshift-pipelines-test/openshift_pipelines_ci.git"
```

Comment:
```suggestion
git clone --branch main "https://github.com/openshift-pipelines-test/openshift_pipelines_ci.git" .
```

Reply:
You want to checkout the repo into the current directory (created and entered in by the above command)

Reply:
Yeah :-/ I added `cd openshift_pipelines_ci`